tests/utils.py

```diff
@@ -162,3 +163,25 @@ def validate_dict(self, d, obj_type):
         except jsonschema.exceptions.ValidationError as e:
             print('Validation error in {}'.format(obj_type))
             raise e
+
+def test_to_from_dict(stac_object_class, d):
+    def _parse_times(dd):
+        a_dict = copy(dd)
```

Comment:
Unclear what the copy/return value do here, since it seems to rely on in-place mutation of the dict (and sub-dictionaries are not reset into the parent copies).

Reply:
yeah, you're right on the return value. I copied the dict in `_parse_times` for the purpose of the input dict 'd'. Otherwise it will modify d and make it so that you can't reuse it in the test function, which you may want to do.

Reply:
and then of course I guess if I include the copy then I need the return